tests/spread/integration/openjdk-25-jdk-headless/task.yaml

```diff
@@ -0,0 +1,128 @@
+        # /usr/lib/jvm/java-25-openjdk-*/bin/javap:
+        chroot . ${home}/javap -l /Main.class
+        # /usr/lib/jvm/java-25-openjdk-*/bin/jdeprscan:
+        chroot . ${home}/jdeprscan --class-path . Main
+        # /usr/lib/jvm/java-25-openjdk-*/bin/jdeps:
+        chroot . ${home}/jdeps -m java.base
+        # /usr/lib/jvm/java-25-openjdk-*/bin/jimage:
+        chroot . ${home}/jimage info ${home}/../lib/modules
+        # /usr/lib/jvm/java-25-openjdk-*/bin/serialver:
+        chroot . ${home}/serialver -classpath / SerializableObject
+      done
+      ;;
+    standard)
+      for java in `find usr/lib/jvm -name java`; do
+        home=$(dirname ${java})
+        # /usr/lib/jvm/java-25-openjdk-*/bin/jar:
+        # /usr/lib/jvm/java-25-openjdk-*/bin/jarsigner:
+        chroot . ${home}/jar cvf test.jar *.java
+        DNAME="CN=Sample Cert, OU=R&D, O=Company Ltd., L=Dublin 4, S=Dublin, C=IE"
+        chroot . ${home}/keytool -genkeypair -keystore foo -storepass barbar -keyalg RSA -dname "$DNAME" -alias foo
+        chroot . ${home}/jarsigner -keystore foo -storepass barbar test.jar foo
+        # /usr/lib/jvm/java-25-openjdk-*/bin/jdb:
+        chroot . /usr/bin/sh -c "echo run | ${home}/jdb Main.java"
+        # /usr/lib/jvm/java-25-openjdk-*/bin/jcmd:
+        chroot . ${home}/jcmd jdk.compiler/com.sun.tools.javac.launcher.Main VM.version
+        # /usr/lib/jvm/java-25-openjdk-*/bin/jhsdb:
+        if [ -f ${home}/jhsdb ]; then
+          chroot . ${home}/jhsdb jstack --pid ${pid}
+        fi
+        # /usr/lib/jvm/java-25-openjdk-*/bin/jfr:
+        chroot . ${home}/jcmd ${pid} JFR.start maxsize=1MB
+        chroot . ${home}/jcmd ${pid} JFR.stop
+        chroot . ${home}/jcmd ${pid} JFR.dump filename=/tmp/recording.jfr
+        chroot . ${home}/jfr print /tmp/recording.jfr > /dev/null
+        # /usr/lib/jvm/java-25-openjdk-*/bin/jinfo:
+        chroot . ${home}/jinfo ${pid}
+        # /usr/lib/jvm/java-25-openjdk-*/bin/jshell:
+        chroot . /usr/bin/sh -c "echo 'System.out.println(\"hello world\")' | ${home}/jshell"
+        # /usr/lib/jvm/java-25-openjdk-*/bin/jmap:
+        chroot . ${home}/jmap ${pid}
+        # /usr/lib/jvm/java-25-openjdk-*/bin/jps:
+        chroot . ${home}/jps -l
+        # /usr/lib/jvm/java-25-openjdk-*/bin/jstack:
+        chroot . ${home}/jstack ${pid}
+        # /usr/lib/jvm/java-25-openjdk-*/bin/jstat:
+        chroot . ${home}/jstat -gc ${pid}
+        # /usr/lib/jvm/java-25-openjdk-*/bin/jstatd:
+        nohup chroot . ${home}/jstatd > ./jstatd.log &
+        pids+=($!)
+        for retry in 0 1 2 3 4 5; do
+          if [ ${retry} -eq 5 ]; then
+            exit 1
+          fi
+          grep -q "bound to /JStatRemoteHost" "jstatd.log" && break
+          sleep 10
+        done
+        # /usr/lib/jvm/java-25-openjdk-amd64/bin/jwebserver
+        nohup chroot . ${home}/jwebserver &
+        sleep 10
+        pids+=($!)
+        for retry in 0 1 2 3 4 5; do
```

Comment:
there are slightly shorter ways of doing this. see, e.g. https://github.com/canonical/ruby-rock/blob/79ca0043dc3715051b130d32d7a26b19e6092a88/ruby/3.3-25.04/spread.yaml#L21. don't change it here now though. it works